eth/downloader/queue.go

```diff
@@ -326,7 +326,7 @@ func (q *queue) Schedule(headers []*types.Header, from uint64) []*types.Header {
 	// Insert all the headers prioritised by the contained block number
 	inserts := make([]*types.Header, 0, len(headers))
 	for _, header := range headers {
-		// Make sure chain order is honoured and preserved throughout
+		// Make sure chain order is honored and preserved throughout
```

Comment:
What's wrong with "honoured"? You prefer American English over Proper English? Tssk